### DISCUSSION TITLE
Data types in prisma

### DISCUSSION THREAD

**Discussion by perry19** [prisma/prisma#7586] [Q&A] [answered]
Hello everyone  I can't find a list of the data types for Prisma schema

I'm trying to use ENUM type and dateonly in my model but I can't find the keyword...

**Comment by ryands17** [comment, marked as answer]
@perry19 👋 

Enums can be found [here](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#enum) and the other data types for all databases can be found [here](https://www.prisma.io/docs/concepts/database-connectors) for after selecting a database.

**Reply by perry19** [reply]
Hi @ryands17 ,
Thanks I found it

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
